src/App.js

```diff
@@ -4,14 +4,24 @@ import Whiteboard from './components/Whiteboard/whiteboard';
 import Toolbar from './components/Toolbar/toolbar';
 
 function App() {
+  const [choice, setChoice] = useState("pencil");
+
+  const onClickHandler = (e) => {
+    if (e === "pencil")
```

Comment:
Define the map here and get value from map instead of putting conditions. 

Something like. 
```

const elements = {
    "pencil" : "pencil",
    "circle" : "circle",
}


const onClick = (e) => {    
    setChoice(elements[e]);
}

//handle undefine or default value in setChoice()

```